src/scripts/shim.js

```diff
@@ -0,0 +1,81 @@
+IS_SETTINGS=false;
+
+function isDevEnvironment(){
+    let port = location.port;
+    return (port == "9393");
```

Comment:
rather use triple equal to check if port is 9393, like `return (port === 9393);`